docs/en/serving/restful_api.md

```diff
@@ -9,7 +9,7 @@ The user can open the http url print by the following command in a browser.
 - **Please check the http url for the detailed api usage!!!**
 
 ```shell
-lmdeploy serve api_server ./workspace --server-name 0.0.0.0 --server-port ${server_port} --tp 1
+lmdeploy serve api_server ./workspace --server-name 0.0.0.0 --server-port ${server_port} --tp 1  --cache-max-entry-count 0.1 
```

Comment:
may leave it as default. Just remind users that more arguments can be found through `-h`